src/blog/2024/10/exploring-flowfuse-sbom-feature.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How the FlowFuse SBOM Feature Enhances Node-RED Application Security and Management
```

Comment:
A better title that doesn't make it sound like a tutorial or documentation and more like a blog/article would be "FlowFuse SBoM helps enhance Node-RED Application Security and Management" Something like this also invokes curiosity to find out how